documentation/release_notes.markdown

```diff
@@ -10,13 +10,33 @@ canonical: "/puppetserver/latest/release_notes.html"
 [puppetserver.conf]: ./config_file_puppetserver.markdown
 [product.conf]: ./config_file_product.markdown
 
+## Puppet Server 6.10.0
+
+Released 14 April 2020
+
+### New features
+
+- The `GET /certificate_status` endpoint now returns certificate or CSR's authorization extensions. [SERVER-2718](https://tickets.puppetlabs.com/browse/SERVER-2718)
+
+- Puppet's "ppRegCertExt" arc has been extended with OID "1.3.6.1.4.1.34380.1.1.26" and the short name "pp_owner". This OID is meant to help users in cloud environments, and the short name will be displayed when using the `puppetserver ca` cli tool.
```

Comment:
should this use backticks instead of quotes? not really sure if there are conventions around that.

Reply:
also, I don't think a comma belongs in that second sentence. either no comma or make it two sentences, imo.

Reply:
I can do that. I'm not really sure what the conventions are.